pkg/ansible/runner/runner.go

```diff
@@ -244,10 +245,26 @@ func (r *runner) Run(ident string, u *unstructured.Unstructured, kubeconfig stri
 		} else {
 			dc = r.cmdFunc(ident, inputDir.Path)
 		}
+		outBuf := bytes.Buffer{}
+		errBuf := bytes.Buffer{}
+		dc.Stdout = &outBuf
+		dc.Stderr = &errBuf
 
 		err := dc.Run()
```

Comment:
It would probably make logs more clear if we used `dc.CombinedOutput()` here so that stdout and stderr are correctly interleaved.

Reply:
TIL, thanks @joelanford

Reply:
Brilliant. I'll make the switch.